gravitee-apim-console-webui/src/management/api/proxy/properties/properties-add-dialog/properties-add-dialog.component.html

```diff
@@ -18,9 +18,38 @@
 <div class="title">
   <h2 mat-dialog-title>New property</h2>
   <mat-icon class="title__cancel"></mat-icon>
-  <button mat-icon-button [attr.aria-label]="'Close'" [mat-dialog-close]="{}">
+  <button mat-icon-button [attr.aria-label]="'Close'" [mat-dialog-close]="">
     <mat-icon class="title__cancel" svgIcon="gio:cancel"> </mat-icon>
   </button>
 </div>
 
-<mat-dialog-content> 🦊 </mat-dialog-content>
+<mat-dialog-content [formGroup]="formGroup">
+  <mat-form-field class="field">
+    <mat-label>Key</mat-label>
+    <input type="text" aria-label="Key input" matInput formControlName="key" required />
+    <mat-error *ngIf="formGroup.get('key').hasError('required')">Key is required.</mat-error>
+    <mat-error *ngIf="formGroup.get('key').hasError('isUnique')">Key already exists.</mat-error>
+  </mat-form-field>
+
+  <mat-form-field class="field">
+    <mat-label>Value</mat-label>
+    <input type="text" aria-label="Value input" matInput formControlName="value" />
+  </mat-form-field>
+
+  <gio-form-slide-toggle class="field">
+    <gio-form-label>Encrypt</gio-form-label>
+    The value will be encrypted on save then it's not possible to retrieve the value after. To change the value, you will have to renew the
```

Comment:
This sentence sounds a bit strange, specially the last part "To change the value, you will have to renew the value."
Maybe this can be rephrased to 
"You will not be able to change value. Instead you will need to delete it and create a new property" ?

Reply:
maybe juste "The value will be encrypted on save then it's not possible to retrieve the value after. " 
Then the use can see the Renew button on the table 🤷‍♂️

Reply:
Yes. Maybe just
> "The value will be encrypted on save, and cannot be retrieved later on"